jupyter_packaging/setupbase.py

```diff
@@ -66,159 +73,168 @@ def list2cmdline(cmd_list):
 
 
 # ---------------------------------------------------------------------------
-# Public Functions
+# Core Functions
 # ---------------------------------------------------------------------------
 
-def get_version(file, name='__version__'):
-    """Get the version of the package from the given file by
-    executing it and extracting the given `name`.
-    """
-    path = os.path.realpath(file)
-    version_ns = {}
-    with io.open(path, encoding="utf8") as f:
-        exec(f.read(), {}, version_ns)
-    return version_ns[name]
+def wrap_installers(build_func, *, wrap_develop=True, wrap_dist=True):
```

Comment:
```suggestion
def wrap_installers(prebuild_func, *, wrap_develop=True, wrap_dist=True):
```

Reply:
`prebuild_func` might be more clear that this function happens *before* the package build step.